policies/model/src/main/java/org/eclipse/ditto/policies/model/ImmutableResourcePermissions.java

```diff
@@ -0,0 +1,137 @@
+    private final String resourcePath;
+    private final List<String> permissions;
+
+    private ImmutableResourcePermissions(final String resourceType, final String resourcePath, final List<String> permissions) {
+        this.resourceType = resourceType;
+        this.resourcePath = resourcePath;
+        this.permissions = Collections.unmodifiableList(permissions);
+    }
+
+    /**
+     * Returns a new instance of {@code ImmutableResourcePermission} based on the provided {@code resourceType},
+     * {@code resourcePath}, and {@code permissions}.
+     *
+     * @param resourceType the type of the resource.
+     * @param resourcePath the path of the resource (as a String).
+     * @param permissions the permissions associated with this ResourcePermission.
+     * @return a new ResourcePermission.
+     * @throws NullPointerException if any argument is {@code null}.
+     * @throws IllegalArgumentException if {@code resourceType} or {@code resourcePath} is empty.
+     */
+    public static ImmutableResourcePermissions newInstance(final String resourceType, final String resourcePath,
+            final List<String> permissions) {
+        return new ImmutableResourcePermissions(
+                argumentNotEmpty(resourceType, "resourceType"),
+                argumentNotEmpty(resourcePath, "resourcePath"),
+                checkNotNull(permissions, "permissions")
+        );
+    }
+
+    /**
+     * Creates an {@code ImmutableResourcePermission} instance from the given JSON object without creating a {@code ResourceKey}.
+     * The resourceKey will be created later by the actor using the provided attributes.
+     *
+     * @param jsonObject the JSON object containing the resource permission data.
+     * @return an {@code ImmutableResourcePermission} instance.
+     * @throws DittoJsonException if the JSON object is not valid or missing required fields.
+     */
+    public static ImmutableResourcePermissions fromJson(final JsonObject jsonObject) {
+        final String resourceType = jsonObject.getValueOrThrow(JsonFieldDefinition.ofString("resourceType"));
+
+        final String resourcePath = jsonObject.getValueOrThrow(JsonFieldDefinition.ofString("resourcePath"));
+
+        final List<String> permissions = jsonObject.getValueOrThrow(JsonFactory.newJsonArrayFieldDefinition("hasPermissions"))
+                .stream().map(JsonValue::asString)
+                .collect(Collectors.toList());
+
+        return newInstance(resourceType, resourcePath, permissions);
+    }
+
+    @Override
+    public String getResourceType() {
+        return resourceType;
+    }
+
+    @Override
+    public String getResourcePath() {
+        return resourcePath;
+    }
+
+    @Override
+    public List<String> getPermissions() {
+        return permissions;
+    }
+
+    @Override
+    public JsonObject toJson() {
+        JsonObjectBuilder jsonBuilder = JsonFactory.newObjectBuilder();
+        jsonBuilder.set("resourceType", resourceType);
```

Comment:
Instead of re-defining the strings again, re-use the (class private) defined constants also to-be-used in `fromJson` method.